src/pg/sql/42_observatory_exploration.sql

```diff
@@ -181,6 +181,94 @@ BEGIN
 END
 $$ LANGUAGE plpgsql;
 
+CREATE OR REPLACE FUNCTION cdb_observatory._OBS_GetNumerators(
+  bounds GEOMETRY DEFAULT NULL,
+  section_tags TEXT[] DEFAULT NULL,
+  subsection_tags TEXT[] DEFAULT NULL,
+  other_tags TEXT[] DEFAULT NULL,
+  ids TEXT[] DEFAULT NULL,
+  name TEXT DEFAULT NULL,
+  denom_id TEXT DEFAULT NULL,
+  geom_id TEXT DEFAULT NULL,
+  timespan TEXT DEFAULT NULL
+) RETURNS TABLE (
+  numer_id TEXT,
+  numer_name TEXT,
+  numer_description TEXT,
+  numer_weight NUMERIC,
+  numer_license TEXT,
+  numer_source TEXT,
+  numer_type TEXT,
+  numer_aggregate TEXT,
+  numer_extra JSONB,
+  numer_tags JSONB,
+  valid_denom BOOLEAN,
+  valid_geom BOOLEAN,
+  valid_timespan BOOLEAN
+) AS $$
+DECLARE
+  where_clause_elements TEXT[];
+  geom_clause TEXT;
+  where_clause TEXT;
+BEGIN
+  section_tags := COALESCE(section_tags, (ARRAY[])::TEXT[]);
+  subsection_tags := COALESCE(subsection_tags, (ARRAY[])::TEXT[]);
+  other_tags := COALESCE(other_tags, (ARRAY[])::TEXT[]);
+  ids := COALESCE(ids, (ARRAY[])::TEXT[]);
+  name := COALESCE(name, '');
+  denom_id := COALESCE(denom_id, '');
+  geom_id := COALESCE(geom_id, '');
+  timespan := COALESCE(timespan, '');
```

Comment:
Why do all the previous coalesces instead of just setting the desired as the defaults in the function signature?